torchdynamo/utils.py

```diff
@@ -292,12 +340,12 @@ def torchscript(model, example_inputs, verbose=True):
         return torch.jit.trace(model, example_inputs)
     except Exception:
         if verbose:
-            log.exception("jit error")
+            log_error("jit error")
```

Comment:
`log.exception()` includes a stack trace.   Are you removing that intentionally?

Reply:
No, I missed that this doesn't get raised further, I'll use log.exception